Jagati.cmake

```diff
@@ -69,7 +69,7 @@ if(JagatiVersion)
     message(STATUS "Already loaded Jagati version '${JagatiVersion}', not loading again.")
     return()
 else(JagatiVersion)
-    set(JagatiVersion "0.30.4")
+    set(JagatiVersion "0.30.5")
```

Comment:
I believe line 72 changes the JagatiVersion to 0.30.5

Reply:
Yup, and this version correlates to URLs in the CMakeLists.txt. So any piece of software using the Jagati decides which version it gets.